changelogs/fragments/ovirt_vm_check_next_run_configuration_update_if_exist.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - ovirt_vm: Check next_run configuration update if exist (https://github.com/ansible/ansible/pull/47282/).
```

Comment:
The problem in the changelog is the ": ".  In yaml, that has a special meaning, the separator between a dictionary's key and value elements.

The easiest fix is to change the ": " into a " - ".